app/controllers/static_pages_controller.rb

```diff
@@ -0,0 +1,10 @@
+class StaticPagesController < ApplicationController
+  def home
+  end
```

Comment:
```suggestion
  def home; end
```

Please check this option, I don't remember very clearly =))